datm/atm_comp_nuopc.F90

```diff
@@ -294,15 +295,15 @@ subroutine InitializeAdvertise(gcomp, importState, exportState, clock, rc)
     if (ChkErr(rc,__LINE__,u_FILE_u)) return
     call ESMF_VMBroadcast(vm, anomaly_forcing, CL*8, main_task, rc=rc)
     if (ChkErr(rc,__LINE__,u_FILE_u)) return
-    call ESMF_VMBroadcast(vm, model_meshfile, CL, main_task, rc=rc)
+    call ESMF_VMBroadcast(vm, model_meshfile, CX, main_task, rc=rc)
     if (ChkErr(rc,__LINE__,u_FILE_u)) return
-    call ESMF_VMBroadcast(vm, model_maskfile, CL, main_task, rc=rc)
+    call ESMF_VMBroadcast(vm, model_maskfile, CX, main_task, rc=rc)
     if (ChkErr(rc,__LINE__,u_FILE_u)) return
     call ESMF_VMBroadcast(vm, factorFn_data, CL, main_task, rc=rc)
     if (ChkErr(rc,__LINE__,u_FILE_u)) return
     call ESMF_VMBroadcast(vm, factorFn_mesh, CL, main_task, rc=rc)
     if (ChkErr(rc,__LINE__,u_FILE_u)) return
-    call ESMF_VMBroadcast(vm, restfilm, CL, main_task, rc=rc)
+    call ESMF_VMBroadcast(vm, restfilm, CX, main_task, rc=rc)
```

Comment:
It looks like you changed the length for the broadcast here but didn't change the length of the variable itself. Which is correct? Elsewhere, you changed the length of restfilm, so I assume you meant to change it here?